cmd/ui/src/App.tsx

```diff
@@ -34,17 +34,20 @@ import { unstable_HistoryRouter as BrowserRouter, useLocation } from 'react-rout
 import Header from 'src/components/Header';
 import { initialize } from 'src/ducks/auth/authSlice';
 import { ROUTE_EXPIRED_PASSWORD, ROUTE_LOGIN, ROUTE_USER_DISABLED } from 'src/ducks/global/routes';
-import { featureFlagKeys, getFeatureFlags } from 'src/hooks/useFeatureFlags';
+import { useFeatureFlags } from 'src/hooks/useFeatureFlags';
```

Comment:
we also have `useFeatureFlag('dark_mode')` though I'm not sure if that will warm up the cache in the same way

Reply:
I think it does, it uses the same query key

Reply:
yeah it hits the same api in the end! I figured surface all feature flags here for future use cases